EventFilter/Phase2TrackerRawToDigi/test/Phase2TrackerDumpDigi_cfg.py

```diff
@@ -0,0 +1,77 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
All jobs should have a comment at the top explaining what they do. The file name suggests that this one dumps the digis. But does it really dump clusters?